python/tvm/tir/ir_builder.py

```diff
@@ -103,7 +103,7 @@ def __getitem__(self, index):
         index = self._linear_index(index)
         if t.lanes > 1:
             base = index * t.lanes
-            index = _expr.Ramp(base, const(1, base.dtype), t.lanes)
+            index = _expr.Ramp(base, 1, t.lanes)
```

Comment:
Probably best as a separate review with a failing/fixed test case.

Reply:
I just uploaded the PR:  https://github.com/apache/incubator-tvm/pull/6941